frontend/src/components/NormalizedConversation/PendingApprovalEntry.tsx

```diff
@@ -129,23 +130,26 @@ function DenyReasonForm({
   onCancel,
   onSubmit,
   inputRef,
+  projectId,
 }: {
   isResponding: boolean;
   value: string;
   onChange: (v: string) => void;
   onCancel: () => void;
   onSubmit: () => void;
   inputRef: React.RefObject<HTMLTextAreaElement>;
+  projectId?: string;
 }) {
   return (
     <div className="mt-3 bg-background px-3 py-3 text-sm">
-      <Textarea
+      <FileSearchTextarea
         ref={inputRef}
         value={value}
-        onChange={(e) => onChange(e.target.value)}
-        placeholder="Let the agent know why this request was denied..."
+        onChange={onChange}
+        placeholder="Let the agent know why this request was denied... (type @ to search files)"
```

Comment:
```suggestion
        placeholder="Let the agent know why this request was denied... Type @ to search files."
```

Match the text in the follow up box